context/design-tokens/alias/default/typography/typography.json

```diff
@@ -81,6 +81,20 @@
 					"value": "{font-weight.bold.value}"
 				}
 			}
+		},
```

Comment:
😄

Reply:
@sturobson  I don't get it 😅 did I put them in the wrong place?

Reply:
sorry, this was just a "yay, more tokens!" smile 😄